trans/views/translation.py

```diff
@@ -354,13 +360,13 @@ def post(self, request, contest_id):
             if settings.MONITOR_ADDRESS:
                 try:
                     response = requests.get(settings.MONITOR_ADDRESS + '/status/printfinal/' + user.username)
-                    response = requests.get('{}/extra?countrycode={}&extra1={}&extra2={}'.format(settings.MONITOR_ADDRESS, user.username, extra_country1, extra_country2))
+                    response = requests.get('{}/extra?countrycode={}&extra1={}&extra2={}'.format(settings.MONITOR_ADDRESS, user.username, extra_country1, extra_country2, extra_country1_count, extra_country2_count))
```

Comment:
I don't think we need to modify this line; we are not using monitor address anymore